terraform/operations/terraform.tfvars

```
@@ -0,0 +1,5 @@
+# tf variables declarations
+project          = "way-lab-saguaro-timelapse"
+region           = "us-central1"
+bucket_name      = "gc-saguaro_udm_bucket"
+initiative_label = "gc-saguaro_udm_bucket"
```

These seemed to have formatting issues when it came to naming expectations (I ran into the error: `Error: "account_id" ("gc-saguaro_udm_bucket-svc-acct") doesn't match regexp "^[a-z](?:[-a-z0-9]{4,28}[a-z0-9])$"`).
```suggestion
bucket_name      = "gc-saguaro-udm-bucket"
initiative_label = "gc-saguaro-udm-bucket"
```